indigo_app/templates/indigo_api/_task_filter_form.html

```diff
@@ -80,5 +80,9 @@
         </div>
       </div>
     {% endif %}
+
+  <!-- Hidden field to capture and preserve the taxonomy_topic parameter-->
+  <input type="hidden" name="taxonomy_topic" value="{{ request.GET.taxonomy_topic }}" />
```

Comment:
I think you can also just do `form.taxonomy_topic.as_hidden`

Reply:
this is the more form-oriented approach, since `request.GET` bypasses the form completely.

Reply:
@longhotsummer We need to set the value from request.GET.

Reply:
shouldn't the form already have it as a value? The form is created based on `request.GET`

Reply:
Ah, right. Its working. Thanks